endpointmanager/pkg/endpointmanager/fhirendpointinfo_test.go

```diff
@@ -435,6 +505,29 @@ func Test_FHIREndpointInfoEqual(t *testing.T) {
 	}
 	endpointInfo2.OperationResource = endpointInfo1.OperationResource
 
+	endpointInfo1.SupportedProfiles[0] = SupportedProfile{
+		Resource:    "",
+		ProfileURL:  "http://hl7.org/fhir/StructureDefinition/daf-allergyintolerance",
+		ProfileName: "Wrong Profile Name",
+	}
+
+	if endpointInfo1.Equal(endpointInfo2) {
+		t.Errorf("Did not expect endpointInfo1 to equal endpointInfo 2. SupportedProfiles should be different. %+v vs %+v", endpointInfo1.SupportedProfiles[0], endpointInfo2.SupportedProfiles[0])
+	}
+	endpointInfo1.SupportedProfiles = endpointInfo2.SupportedProfiles
+
+	endpointInfo2.SupportedProfiles = make([]SupportedProfile, 0)
+	if endpointInfo1.Equal(endpointInfo2) {
+		t.Errorf("Did not expect endpointInfo1 to equal endpointInfo 2. SupportedProfiles should be different. %+v vs %+v", endpointInfo1.SupportedProfiles, endpointInfo2.SupportedProfiles)
+	}
+	endpointInfo2.SupportedProfiles = endpointInfo1.SupportedProfiles
+
+	endpointInfo1.SupportedProfiles = make([]SupportedProfile, 0)
+	if endpointInfo1.Equal(endpointInfo2) {
+		t.Errorf("Did not expect endpointInfo1 to equal endpointInfo 2. SupportedProfiles should be different. %+v vs %+v", endpointInfo1.SupportedProfiles, endpointInfo2.SupportedProfiles)
+	}
+	endpointInfo1.SupportedProfiles = endpointInfo2.SupportedProfiles
```

Comment:
I'd also just write a check where they do equal each other, just to cover all of our bases!